backend/src/plugins/ModActions/commands/CasesModCmd.ts

```diff
@@ -28,8 +28,10 @@ export const CasesModCmd = modActionsCmd({
 
   async run({ pluginData, message: msg, args }) {
     const modId = args.mod || msg.author.id;
-    const mod = await resolveUser(pluginData.client, modId);
-    const modName = mod instanceof User ? mod.tag : modId;
+    const mod =
+      (await resolveMember(pluginData.client, pluginData.guild, modId)) ||
+      (await resolveUser(pluginData.client, modId));
+    const modName = mod instanceof User ? renderUsername(mod) : modId;
```

Comment:
This doesn't account for GuildMember, maybe check the opposite instead (you have to import it)
```suggestion
    const modName = mod instanceof UnknownUser ? modId : renderUsername(mod);
```